Makefile

```diff
@@ -56,16 +55,13 @@ WINDOWS_BUILDDIR=$(BUILDDIR)/windows
 
 windows: $(WINDOWS_BUILDDIR)/tun2socks.exe
 
-$(WINDOWS_BUILDDIR)/tun2socks.exe: $(XGO)
-	mkdir -p "$(WINDOWS_BUILDDIR)/$(IMPORT_PATH)"
-	$(XGO) -ldflags $(XGO_LDFLAGS) --targets=windows/386 -dest "$(WINDOWS_BUILDDIR)" -pkg $(ELECTRON_PKG) .
-	mv "$(WINDOWS_BUILDDIR)/$(IMPORT_PATH)-windows-386.exe" "$@"
-	rm -r "$(WINDOWS_BUILDDIR)/$(IMPORT_HOST)"
-
+$(WINDOWS_BUILDDIR)/tun2socks.exe:
+	mkdir -p "$(@D)"
+	GOOS=windows GOARCH=amd64 CGO_ENABLED=1 CC="x86_64-w64-mingw32-gcc" go build --ldflags=--extldflags=$(XGO_LDFLAGS) -o "$@" $(ELECTRON_PKG)
```

Comment:
Maybe not in this PR: our CI is not checking the build file (it is building directly from native Windows), we may need to add some x-compile CI jobs in the future.